functions.py

```diff
@@ -13,44 +14,64 @@ def search(path, primary_key):
 
 def creates_dir(schema):
     x = json.load(open(schema, 'r'))
-    p = os.path.join(os.getcwd(), x['database_name'])
-    os.mkdir(p)
-    for i in x['Tables']:
-        d = os.path.join(p, i['name'])
-        os.mkdir(d)
+    p = os.path.join(os.getcwd(), x[keys().database])

```

Comment:
You don’t need to check if directory exists or not.
MakeDir take a Boolean true. When we pass this Boolean, that means don’t do anything if directory exist

Reply:
That is the next command because the database could be created already but another could be added in some time so it will check that every table is created even if the database itself was created

Reply:
Lets' refactor the code and follow the design pattern, then we can return to this later.